### DISCUSSION TITLE
Codespace environment working only 15 minutes

### DISCUSSION THREAD

**Discussion by helloivankorGit** [community/community#102688] [Codespaces]
### Select Topic Area

Bug

### Body

The port forwarding is added but the frontend is not available with the following error (screenshot).
Been trying to restart, recreate the containers and also use everything in the browser but no luck. The codespace is spinning up, the ports to forward are added (80) but in the browser the error (from browser) is shown. I need to disable my WIFI connection, waiting 5 minutes, enable WIFI - codespace will work. But only for 15 minutes. And so on, again and again 

![image (3)](https://github.com/community/community/assets/72932438/7e4ad1df-ca44-43c0-8801-4d6edf1b5f03)

**Comment by jonathanmadeley** [comment]
Recently experienced this same issue. Solution was to increase the size of the machine on Codespaces. Haven't had the issue since.
